libaleth/SendDialog.ui

```diff
@@ -2,6 +2,32 @@
 <ui version="4.0">
  <class>SendDialog</class>
  <widget class="QDialog" name="SendDialog">
+  <property name="geometry">
+   <rect>
+    <x>0</x>
+    <y>0</y>
+    <width>223</width>
```

Comment:
you are right there is a small gap. but the question for AlethOne is still valid :)